src/client/opamSolution.ml

```diff
@@ -1228,17 +1228,18 @@ let install_depexts ?(force_depext=false) ?(confirm=true) t packages =
       map_sysmap (fun _ -> OpamSysPkg.Set.empty) t
     with Failure msg ->
       OpamConsole.error "%s" msg;
-      manual_install ~noninteractive:`Quit t sys_packages
+      check_again t sys_packages
   and check_again t sys_packages =
-    let needed, _notfound = OpamSysInteract.packages_status sys_packages in
+    let needed, _notfound = OpamSysInteract.packages_status ~env sys_packages in
```

Comment:
This one was well hidden, but at least one test caught it. Maybe the parameter shouldn't have been optional :grimacing:

Reply:
Totally agree! Optional parameters should be avoided as much as possible IMO.